EXPhotoViewer/EXPhotoViewer.m

```diff
@@ -58,6 +58,7 @@ -(void)loadView {
     UIImageView* imageView = [[UIImageView alloc] initWithFrame:self.view.bounds];
     imageView.contentMode = UIViewContentModeScaleAspectFill;
```

Comment:
whoops... Pushed...

Thank you

On Wed, Jan 28, 2015 at 10:25 PM, Spencer Comerford <
notifications@github.com> wrote:

> In EXPhotoViewer/EXPhotoViewer.m
> <https://github.com/ShowMobile/EXPhotoViewer/pull/1#discussion_r23720195>:
>
> > @@ -58,6 +58,7 @@ -(void)loadView {
> >      UIImageView* imageView = [[UIImageView alloc] initWithFrame:self.view.bounds];
> >      imageView.contentMode = UIViewContentModeScaleAspectFill;
>
> Just remove this line (its unnecessary now) and I'll merge this in.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ShowMobile/EXPhotoViewer/pull/1/files#r23720195>.
>